src/test/java/pl/marcin/project/repository/CustomerRepositoryListBasedTest.java

```diff
@@ -38,7 +40,166 @@ public void shouldReturnCustomerWithId2() {
+    }
+
+    @Test
+    public void shouldReturnTrueIfDeleted() {
+        //given
+        int id = 1;
+        Customer customer1 = Customer.builder().id(id).build();
+        customerRepository.saveCustomer(customer1);
+        //when
+        boolean ifDeleted = customerRepository.deleteCustomer(1);
+        //then
+        Assertions.assertTrue(ifDeleted);
+    }
+
+    @Test
+    public void shouldReturnFalseIfNotDeleted() {
+        //given
+        int id = 1;
+        Customer customer1 = Customer.builder().id(id).build();
+        customerRepository.saveCustomer(customer1);
+        //when
+        boolean ifDeleted = customerRepository.deleteCustomer(2);
+        //then
+        Assertions.assertFalse(ifDeleted);
+    }
+
+    @Test
+    public void shouldReturnCustomerId() {
+        //given
+        int id = 1;
+        Customer customer1 = Customer.builder()
+                .id(id)
+                .name("Jan")
+                .surname("Kowalski")
+                .address("Kraków")
+                .build();
+        //when
+        int returnedId = customerRepository.saveCustomer(customer1);
+        //then
+        Assertions.assertEquals(id, returnedId);
+    }
+
+    @Test
+    public void shouldEqualToSavedCustomer() {
+        //given
+        int id = 1;
+        Customer customer1 = Customer.builder()
+                .id(id)
+                .name("Jan")
+                .surname("Kowalski")
+                .address("Kraków")
+                .build();
+        //when
+        customerRepository.saveCustomer(customer1);
+        //then
+        Assertions.assertEquals(customer1, customerRepository.findCustomer(1));
+    }
 
+    @Test
+    public void shouldEqualToUpdatedCustomer() {
+        //given
+        int id = 1;
+        Customer customer1 = Customer.builder().id(id).build();
+        customerRepository.saveCustomer(customer1);
+
+        //when
+        Customer updatedCustomer = Customer.builder()
+                .id(1)
+                .name("Jacek")
+                .surname("Placek")
+                .address("Pułtusk")
+                .build();
+        customerRepository.updateCustomer(id, updatedCustomer);
+        //then
+        Assertions.assertEquals(updatedCustomer, customerRepository.findCustomer(id));
+    }
+
+    @Test
+    public void shouldThrowException() {
+        //given
+        int id = 1;
+        Customer customer1 = Customer.builder().id(id).build();
+        customerRepository.saveCustomer(customer1);
+
+        //when
+        Customer updatedCustomer = Customer.builder()
+                .id(2)
+                .name("Jacek")
+                .surname("Placek")
+                .address("Pułtusk")
+                .build();
+
+        //then
+        assertThrows(RuntimeException.class, () -> {
+            customerRepository.updateCustomer(2, updatedCustomer);
+        });
+    }
+
+    @Test
+    public void shouldReturnUpdatedCustomerId() {
```

Comment:
this test with this one `shouldEqualToUpdatedCustomer` can be merged

Reply:
ok